examples/2d/tilemap_chunk.rs

```diff
@@ -27,10 +27,16 @@ fn setup(mut commands: Commands, assets: Res<AssetServer>) {
     let mut rng = ChaCha8Rng::seed_from_u64(42);
 
     let chunk_size = UVec2::splat(64);
-    let tile_display_size = UVec2::splat(8);
-    let indices: Vec<Option<u16>> = (0..chunk_size.element_product())
+    let tile_display_size = UVec2::splat(16);
```

Comment:
I think 8 was a nicer size since the entire chunk can fit in the screen at that size